gadopt/utility.py

```diff
@@ -515,3 +516,50 @@ def interpolate_1d_profile(function: Function, one_d_filename: str):
     averager = LayerAveraging(mesh, rshl if mesh.layers is None else None)
     interpolated_visc = np.interp(averager.get_layer_average(rad), rshl, one_d_data)
     averager.extrapolate_layer_average(function, interpolated_visc)
+
+
+def get_boundary_ids(mesh) -> SimpleNamespace:
```

Comment:
Worthwhile to also unit test this against all the (feasible) Firedrake meshes? What if we give this e.g. a gmsh mesh?

Reply:
I thought about that but don't really know the best way to go about doing that. Firedrake can make a lot of different meshes.